python/taichi/misc/gui.py

```diff
@@ -210,7 +210,12 @@ def set_image(self, img):
     def circle(self, pos, color=0xFFFFFF, radius=1):
         self.canvas.circle_single(pos[0], pos[1], color, radius)
 
-    def circles(self, pos, color=0xFFFFFF, radius=1):
+    def circles(self,
+                pos,
+                color=0xFFFFFF,
```

Comment:
nit: Move `color` above `color`, so that all color-related things are consecutive?

Reply:
I'm not sure what this means?

Reply:
Sorry, I meant to move `radius` above `color`...